Content.Server/Fluids/EntitySystems/DrainSystem.cs

```diff
@@ -24,27 +30,33 @@ public override void Update(float frameTime)
                 }
                 drain.Accumulator -= drain.DrainFrequency;
 
-                /// Best to do this one every second rather than once every tick...
-                _solutionSystem.TryGetSolution(drain.Owner, DrainComponent.SolutionName, out var drainSolution);
+                if (!managerQuery.TryGetComponent(drain.Owner, out var manager))
+                    continue;
```

Comment:
just put it in the `EntityQuery`?

Reply:
Seemed better considering 90% of the time the float is just getting accumulated and we don't need the solutioncontainermanager

Reply:
i suppose so